docker-compose.yml

```diff
@@ -0,0 +1,18 @@
+services:
+  postgis:
+    image: postgis/postgis:15-3.3
+    env_file:
+      - ./secrets/.env
+    ports:
+      - "5432:5432"
+    restart: unless-stopped
+    volumes:
+      - database:/var/lib/postgresql/data
+      - ./database/create_tables.sql:/docker-entrypoint-initdb.d/create_tables.sql
```

Comment:
Nice !

Reply:
why not puuting the create table also here ? therefore we can just run the docker-compose...